src/pages/guide/language/list.md

```diff
@@ -8,6 +8,12 @@ Lists are homogeneous, immutable, and support fast `O(1)` append at the head of
 ```reason
 let myList = [1, 2, 3];
 let anotherList = [0, ...myList]; /* myList didn't mutate */
+let listOfRecords = [
+    { 
+        title: "new record"
```

Comment:
hmmm this might be confusing b/c we don't define the record type or oldRecords (would be best to have the code snippet be 100% valid).
Could you do maybe 
```
type thing = {title: string};
let oldRecords = [{title: "one"}];
let listOfRecords = [{title: "two"}, ...oldRecords]
```